test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -900,27 +908,59 @@ UCS_TEST_P(test_uct_cm_sockaddr, connect_client_to_server_reject_with_delay)
     test_delayed_server_response(true);
 }
 
-UCS_TEST_P(test_uct_cm_sockaddr, double_ep_disconnect)
+UCS_TEST_P(test_uct_cm_sockaddr, ep_disconnect_err_codes)
 {
-    ucs_status_t status;
+    bool disconnecting = false;
 
     cm_listen_and_connect();
 
+    m_client->async_block();
+        if (m_cm_state & TEST_CM_STATE_CLIENT_CONNECTED) {
+            UCS_TEST_MESSAGE << "EXP: " << ucs_status_string(UCS_OK);
```

Comment:
why need all those EXP print?

Reply:
this test has more 1 scenario depends on timings, expected status prints might be helpful for reproducibility/debugging in case of failure in CI